packages/closed_household/lib/pages/closed_household_details.dart

```diff
@@ -101,23 +102,19 @@ class ClosedHouseholdDetailsPageState
                   builder: (context, locationState) {
                     return DigitElevatedButton(
                       onPressed: () {
+                        final String householdHeadName = form.control(_householdHeadNameKey).value.toString().trim();
```

Comment:
Why do we need to initialize householdHeadName here? We can directly assign the value from form control.

Reply:
Because i am trimming the name like if they just put space in the start or end so after trimming i am assigning that value